testdata/p4_16_errors/header2_e.p4

```diff
@@ -1,34 +0,0 @@
-/*
-Copyright 2016 VMware, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-struct s {}
-
-header h
-{
-    s field;  // no struct fields allowed in headers
-}
-
-parser p();
-
-struct s1
-{
-    p field;  // no functor-typed fields allowed
```

Comment:
this negative test should probably remain. it is a valid test.

Reply:
Agree.

Reply:
Why is this marked as resolved? The file still seems to be deleted.
Please don't mark conversations as resolved. If you make the requested changes github will mark them automatically.

Reply:
Hmm, I checked the PR workspace <https://github.com/hesingh/p4c-1> and do see this file in the path.  Thus the file is not deleted.   I will double-check this file is run during 'make check.